client/my-sites/plans/jetpack-plans/product-store/hooks/use-store-item-info.tsx

```diff
@@ -58,9 +61,26 @@ export const useStoreItemInfo = ( {
 	const isCurrentUserPurchaseOwner = useIsUserPurchaseOwner();
 	const translate = useTranslate();
 
+	const getProductIsInCart = useCallback(
+		( item: SelectorProduct ) => {
+			const cartProducts = cartManagerClient.forCartKey( siteId || undefined ).getState()
+				.responseCart.products;
+			for ( const product of cartProducts ) {
+				if ( product.product_slug === item.productSlug ) {
+					return true;
+				}
+			}
+			return false;
+		},
+		[ siteId ]
+	);
+
 	// Determine whether product is owned.
 	const getIsOwned = useCallback(
 		( item: SelectorProduct ) => {
+			if ( shouldShowCart && getProductIsInCart( item ) ) {
+				return true;
+			}
```

Comment:
I'm not sure why we would want to show the product as _owned_ ("Active on your site") when the product is in the cart?  I know this is just a draft POC, I'm just leaving the comment here so we can be sure to address it in the real project implementation. 😉